CHANGES.md

```diff
@@ -168,6 +168,12 @@ OpenSSL 3.3
 
    *Alexandr Nedvedicky*
 
+ * Added Attribute Certificate (RFC 5755) support. Attribute
```

Comment:
This needs to move up to `OpenSSL 3.4` paragraph, while rebasing to  last master who recently jump to 3.4